mirrord-cli/src/main.rs

```diff
@@ -108,6 +108,12 @@ fn exec(args: &ExecArgs) -> Result<()> {
         std::env::set_var("MIRRORD_AGENT_NAMESPACE", namespace.clone());
     }
     if let Some(log_level) = &args.agent_log_level {
+        let levels = vec!["trace", "debug", "info", "warn", "error"];
```

Comment:
I'm not sure if we should restrict that, as the rust env logger has very complex configurations you can use.. I think leaving it free text is better